### PR TITLE
fix: longer axios timeout in integ-test

### PR DESCRIPTION
**Pull request by ConjunctiveNormalForm**

### CODE REVIEW

File: lib/providers/order/uniswapxService.ts [hunk 1]
```diff
@@ -18,8 +18,9 @@ export class UniswapXServiceProvider implements OrderServiceProvider {
     this.log.info({ orderHash: order.hash() }, 'Posting order to UniswapX Service');
 
     const axiosConfig = {
```
Comment (codyborn) [inline lib/providers/order/uniswapxService.ts:20]: Can you add a comment above line 20 explaining why we need a longer timeout for testing?

**Comment by codyborn** [comment]
I might be missing some context. If this is for axios in the integ-tests, why not update it in the `AxiosUtils` that the integ-tests use?

**Comment by ConjunctiveNormalForm** [comment]
> I might be missing some context. If this is for axios in the integ-tests, why not update it in the `AxiosUtils` that the integ-tests use?

We call param-api in the integ tests, which in turns posts to order service. Updating AxiosUtils only has an effect on the former but not the latter 😢

File: bin/app.ts [hunk 2]
```diff
@@ -243,6 +247,7 @@ envVars['URA_ACCOUNT'] = process.env['URA_ACCOUNT'] || '';
 envVars['BOT_ACCOUNT'] = process.env['BOT_ACCOUNT'] || '';
 envVars['UNISWAP_API'] = process.env['UNISWAP_API'] || '';
 envVars['ORDER_SERVICE_URL'] = process.env['ORDER_SERVICE_URL'] || '';
+envVars['IS_TEST'] = 'true';
```
Comment (codyborn) [inline bin/app.ts:250]: Why is this hardcoded to true? Why not pull from an env var?

**Review by codyborn — APPROVED** [review]
Left some small comments

**Pull request merged**